m4/common/boost.m4

```diff
@@ -107,13 +126,6 @@ AC_ARG_WITH([boost],
    [AS_HELP_STRING([--with-boost=DIR],
                    [prefix of Boost $1 @<:@guess@:>@])])dnl
 AC_ARG_VAR([BOOST_ROOT],[Location of Boost installation])dnl
-# If BOOST_DIR is set and BOOST_ROOT is not, use BOOST_DIR.
```

Comment:
Thanks, I missed that!

I'd actually prefer to remove it, though.  BOOST_ROOT seems to be "standard" enough, and I'm in the middle of remaking modules so it's easy enough to define that in addition to BOOST_DIR.

Your preference?